osu.Game.Tests/Visual/Gameplay/TestSceneSkinEditor.cs

```diff
@@ -214,7 +217,11 @@ public void TestUndoEditHistory()
             });
 
             AddStep("Press undo", () => InputManager.Keys(PlatformAction.Undo));
-            AddAssert("Nothing changed", () => defaultState.SequenceEqual(changeHandler.GetCurrentState()));
+
+            AddAssert("Nothing changed",
+                () => JsonConvert.DeserializeObject<IEnumerable<SerialisedDrawableInfo>>(Encoding.UTF8.GetString(defaultState)),
+                () => Is.EqualTo(JsonConvert.DeserializeObject<IEnumerable<SerialisedDrawableInfo>>(Encoding.UTF8.GetString(changeHandler.GetCurrentState())))
```

Comment:
This can't ever work because `SerialisedDrawableInfo` doesn't implement `IEquatable<SerialisedDrawableInfo>`. So it will always be a reference comparison that will always fail.

Reply:
Reverted as per IRL discussion